packages/server/src/adapters/aws-lambda/getPlanner.ts

```diff
@@ -1,11 +1,36 @@
+import { Readable, type Writable } from 'node:stream';
+import { pipeline } from 'node:stream/promises';
 import type {
   APIGatewayProxyEvent,
   APIGatewayProxyEventV2,
   APIGatewayProxyResult,
   APIGatewayProxyStructuredResultV2,
+  Context,
 } from 'aws-lambda';
 import { splitSetCookieString } from '../../vendor/cookie-es/set-cookie/split';
 
+declare global {
+  namespace awslambda {
+    class HttpResponseStream extends Writable {
+      static from(
+        writable: Writable,
+        metadata: Record<string, unknown>,
+      ): HttpResponseStream;
+      setContentType: (contentType: string) => void;
+    }
+
+    type StreamifyHandler<TEvent = any, TResult = any> = (
+      event: TEvent,
+      responseStream: awslambda.HttpResponseStream,
+      context: Context,
+    ) => TResult | Promise<TResult>;
+
+    function streamifyResponse<TEvent = any, TResult = void>(
+      handler: StreamifyHandler<TEvent, TResult>,
+    ): StreamifyHandler<TEvent, TResult>;
+  }
+}
+
```

Comment:
Opened PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/72417 to get the types into `aws-lambda` directly

Reply:
PR was merged